javascript/javascript1/week3/distance.js

```diff
@@ -0,0 +1,16 @@
+//When will we be there??
+const travelInformation = {
+    speed: 50,
+    destinationDistance: 432,
+  };
+  
+  function calTime(travelInformation)
+  {
+      let time = (travelInformation.destinationDistance)/(travelInformation.speed);
```

Comment:
Remember to use `const` when not needing to re-assign the variable.

Also you don't need to wrap parenthesis around in this case, the following would be enough:
```js
      const time = travelInformation.destinationDistance / travelInformation.speed;
```

I've added some spacing inbetween to make devide (`/`) stand out.

Reply:
done